autoextract_poet/mixins.py

```diff
@@ -0,0 +1,19 @@
+from web_poet.mixins import ResponseShortcutsMixin
+from web_poet.page_inputs import ResponseData
+
+
+class AutoExtractResponseShortcutsMixin(ResponseShortcutsMixin):
+
```

Comment:
Suggestions wanted.

Reply:
This was a trick to being able to use the HTML returned by AutoExtract the same way that is used in `ResponseData`. I see several problems here:
* HTML won't be always present. We need a way so that the user can configure what he wants. Having a set of new classes (e.g. AEProductPageHTML) just to configure that HTML is required seems explosive. 
* Probably it would be nice to have some kind of `AEResponseData` class for that. So that when you request `AEResponseData` your receive a `ResponseData` compatible class but with HTML coming from AutoExtract. The problem is that requiring both `AEResponseData` and `AEProductPage` should not raise two requests to AutoExtract, but only one. This breaks the current provider structure. We should think about it.

Reply:
What about leaving HTML content out of the scope for a while? I mean, AutoExtract biggest advantage is not having to parse HTML content. We could address it in another ticket, meanwhile, users could use regular responses from the spider callback.

Reply:
Removed for now.